src/OrderFormAcceptanceTests.TestData/Helpers/OrderItemHelper.cs

```diff
@@ -78,6 +78,26 @@ FROM dbo.CataloguePrice
             return result;
         }
 
+        public static async Task<ProvisioningType> GetProvisioningTypeForItemAsync(string itemId, string connectionString)
+        {
+            var query = @"SELECT ProvisioningTypeId
+                            FROM dbo.CataloguePrice
+                            WHERE CataloguePriceId = @itemId;";
+            var result = (await SqlExecutor.ExecuteAsync<ProvisioningType>(connectionString, query, new { itemId })).First();
```

Comment:
Perhaps `.Single` would be a slightly better fit here semantically? As there can't be multiple prices with the same ID.